apps/examples/src/app/basic-smart-dumb-example/basic-smart/basic-smart.component.spec.ts

```diff
@@ -19,12 +19,12 @@ describe('BasicSmartComponent', () => {
     }) as jest.Mocked<RandomBreweryService>;
 
     const injectSpy = jest.spyOn(angularCore, 'inject');
-    injectSpy.mockImplementation((providerToken: unknown) => {
+    injectSpy.mockImplementation(((providerToken: unknown) => {
       if (providerToken === RandomBreweryService) {
         return mockService;
       }
       throw new Error(`Unexpected inject token: ` + providerToken);
-    });
+    }) as any);
```

Comment:
😬

Reply:
Yeah, the inject function now has an additional overload that caused a type error here and I was lazy since it's a test.  But I can do better, so will. :)

Reply:
Used `as typeof angularCore.inject` instead